cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
+#include "cartographer/mapping/sparse_pose_graph.h"
```

Comment:
Style: Related header file comes first and separate from all others, i.e. this belongs one line down, separate by an empty line.